jujugui/static/gui/src/app/init/component-renderers-mixin.js

```diff
@@ -486,7 +486,9 @@ Browser: ${navigator.userAgent}`
         <PostDeployment
           changeState={this._bound.changeState}
           charmstore={shapeup.fromShape(this.charmstore, PostDeployment.propTypes.charmstore)}
-          entityId={entityId} />,
+          entityId={entityId}
+          showPostDeploymentScript={
```

Comment:
I think we can remove this flag as the functionality is already disabled without the post-deployment file in the bundle.

Reply:
I'm not sure I agree, as we may want to push to a public bundle before we enable the flag, allowing us to A/B test.

Reply:
Ok fair!